tiler.Rproj

```diff
@@ -18,3 +18,7 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageRoxygenize: rd,collate,namespace
```

Comment:
Just curious what is the motivation for the new lines in this file.

Reply:
That's just me setting Rstudio project options to auto-generate package docs etc.

Reply:
Ah, thanks!